extensions/src/main/java/org/pentaho/platform/plugin/services/importexport/exportManifest/ExportManifestEntity.java

```diff
@@ -77,9 +91,34 @@ protected ExportManifestEntity( File file, String userId, String projectId, Bool
     createEntityMetaData( file, userId, projectId, isFolder, isHidden, isSchedulable );
     createEntityAcl( userId );
     rawExportManifestProperty.setEntityMetaData( entityMetaData );
+    rawExportManifestProperty.setEntityExtraMetaData( entityExtraMetaData );
     rawExportManifestProperty.setEntityAcl( entityAcl );
   }
 
+  private void createEntityExtraMetaData( RepositoryFile repositoryFile ) {
+    entityExtraMetaData = new EntityExtraMetaData();
+
+    if ( repositoryFile instanceof RepositoryFileProxy ) {
```

Comment:
I understand this solution.
However, I not sure if that has no impact. Namely, if we decide to extends RepositoryFile in new ways...

Reply:
I understand with your concern, but this is a solution it less impact. 
As an alternative solution could be change parent class but I am not sure if metadata have any meaning in RepositoryFile itself.

I am open for suggestion :)